### DISCUSSION TITLE
Stripe connect with subscription

### DISCUSSION THREAD

**Discussion by maulikbafalipara** [pay-rails/pay#672] [General]
Hello All,

Have anyone implement for plans subscription payment flow with stripe connect.

I am want to develop stripe connect with subscription payment but I can't understand how to create plans in merchant's account so if anyone has idea then please let me know,I check direct charge flow and it's working.

**Comment by ockhamlabs** [comment]
@excid3 Would be great to get your thoughts on this

**Reply by excid3** [reply]
Just use the Stripe API directly to create Prices on the account.

**Reply by maulikbafalipara** [reply]
@excid3 Yes, i used Stripe apis to create plans but I want to create plans in the merchant's account so I don't know how to pass merchant account details or something else to create a price.
```
We have a stripe API key and secret key of the main account, not a merchant's account so...
Stripe::Price.create({
  unit_amount: 2000,
  currency: 'usd',
  recurring: {interval: 'month'},
  product: 'prod_MAhkdQGIBR149q',
})
```
Above one right for creating price in stripe.

Using connect we can charge with a merchant account, does it work with subscription charges?

**Reply by excid3** [reply]
See the Stripe docs. You'll supply the `stripe_account` param during your requests. https://github.com/stripe/stripe-ruby#per-request-configuration
